src/gtest/test_keys.cpp

```diff
@@ -126,8 +132,11 @@ TEST(Keys, EncodeAndDecodeUnifiedAddresses)
             // These were added to the UA in preference order by the Python test vectors.
```

Comment:
```suggestion
            // These were added to the UA in typecode order by the Python test vectors.
```

Reply:
I believe we still rely on this order of calls, because #5456 has not yet been closed.